apps/backend/src/app/modules/maps/maps.service.ts

```diff
@@ -539,6 +539,9 @@ export class MapsService {
 
     const bspFile = await this.getBspFromTemp(userID);
 
+    if (bspFile instanceof Error)
```

Comment:
This pattern of returning an `Error` is a bit unusual, I would just return `null`. Returning errors as regular objects isn't *necessarily* a bad thing but confused me a lot at first.

Reply:
Also while we're here let's give this function a clearer name, let's use `tryGetBspFromTempUploads`.